### PR TITLE
[exporter/influxdb] Add logic to validate the config

### LINKED ISSUE #7839: [exporter/influxdb] Ensure Validate implementation does the right thing
This [change](https://github.com/open-telemetry/opentelemetry-collector/commit/48c402dbfafd6c53686a7aad193beb1aa47a4b7c) in the collector core added a `Validate` method to `QueueSettings` which caused ambiguity with the `ExporterSettings`’ `Validate` method. As a result, I’ve added a Validate method to this exporter and an issue to ensure it was doing the right thing.

Here is an example of the validate method for the OTLP exporter calling the QueueSettings’ Validate

https://github.com/open-telemetry/opentelemetry-collector/commit/48c402dbfafd6c53686a7aad193beb1aa47a4b7c#diff-a56037437974f775001642d59880ac9dd0c14cab137576b4f4a6df01a230a84cR39-R42

_Originally posted by @codeboten in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/7779#discussion_r803075118_

### PR DESCRIPTION
**Pull request by DiptoChakrabarty**
Signed-off-by: DiptoChakrabarty <diptochuck123@gmail.com>

**Description:** <Describe what has changed.>

This adds a validate method to ExporterSettings 

**Link to tracking Issue:** <Issue number if applicable>

Closes #7839 

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by DiptoChakrabarty** [comment]
PTAL @jacobmarble @8none1

**Review by jacobmarble — APPROVED** [review]

**Comment by jacobmarble** [comment]
@DiptoChakrabarty thanks for this!

Per [the docs](https://github.com/open-telemetry/opentelemetry-collector/blob/8bcb5f0c229f5ac6555ab3e5ab77646495a562e7/CONTRIBUTING.md#updating-changelog), the changelog should be updated when "Changes made to the behaviour of the component". This change is minor, and wouldn't trigger unexpected behavior. Also, I don't see this check passing in some other PRs, so perhaps this can be ignored.

File: exporter/influxdbexporter/config.go [hunk 1]
```diff
@@ -47,5 +49,8 @@ type Config struct {
 }
 
 func (cfg *Config) Validate() error {
+	if err := cfg.ExporterSettings.Validate(); err != nil {
+		return fmt.Errorf("exporter settings is invalid :%w", err)
```
Comment (Aneurysm9) [inline exporter/influxdbexporter/config.go:53]:
```suggestion
		return fmt.Errorf("exporter settings are invalid :%w", err)
```

**Review by Aneurysm9 — APPROVED** [review]
It appears that this cannot change behavior without a change to `config.ExporterSettings`' `Validate()` method, which would absolutely need a changelog entry.  I think I am ok without a changelog entry for this change.

File: exporter/influxdbexporter/config.go (hunk 1 above)
Comment (DiptoChakrabarty) [reply exporter/influxdbexporter/config.go]: done

**Pull request merged**